Mod Manager/src/routes/modList.svelte

```diff
@@ -84,62 +106,66 @@
 
 	let frameworkModScriptsWarningOpen = false
 
-	async function addMod() {
-		window.ipc.send("modFileOpenDialog")
-
-		window.ipc.receive("modFileOpenDialogResult", (modFilePopupResult: string[] | undefined) => {
-			if (!modFilePopupResult) {
-				return
-			}
+	async function addModNoPopup() {
```

Comment:
You'll probably have to change some of this because of the recent RPKG mod import changes unfortunately, but I definitely like this feature!